frontend/hub/namespaces/HubNamespaces.tsx

```diff
@@ -43,11 +43,13 @@ export function Namespaces() {
 }
 
 export function AllNamespaces() {
-  return <CommonNamespaces url={hubAPI`/_ui/v1/namespaces/`} />;
+  return <CommonNamespaces url="/api/automation-hub/pulp/api/v3/pulp_ansible/namespaces/" />;
 }
 
 export function MyNamespaces() {
-  return <CommonNamespaces url={hubAPI`/_ui/v1/my-namespaces/`} />;
+  return (
+    <CommonNamespaces url="/api/automation-hub/pulp/api/v3/pulp_ansible/namespaces/?my_permissions=ansible.change_ansiblenamespace" />
```

Comment:
```suggestion
    <CommonNamespaces url={pulpAPI`pulp_ansible/namespaces/?my_permissions=ansible.change_ansiblenamespace`} />
```

(but wasn't this `?my_permissions=...` why you had to add `queryParams`?)